src/include/planner/copy_plan.h

```diff
@@ -0,0 +1,59 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// copy_plan.h
+//
+// Identification: src/include/planner/copy_plan.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "planner/abstract_plan.h"
+#include "parser/statement_copy.h"
+#include "parser/statement_select.h"
+
+namespace peloton {
+
+namespace storage {
+class DataTable;
+}
+
+namespace parser {
+class CopyStatement;
+}
+
+namespace planner {
+class CopyPlan : public AbstractPlan {
+ public:
+  CopyPlan() = delete;
+  CopyPlan(const CopyPlan &) = delete;
+  CopyPlan &operator=(const CopyPlan &) = delete;
+  CopyPlan(CopyPlan &&) = delete;
+  CopyPlan &operator=(CopyPlan &&) = delete;
+
+  explicit CopyPlan(parser::CopyStatement *parse_tree);
+
+  inline PlanNodeType GetPlanNodeType() const { return PLAN_NODE_TYPE_COPY; }
+
+  void SetParameterValues(UNUSED_ATTRIBUTE std::vector<common::Value> *values) {
```

Comment:
I think we still want to call SetParameterValues on child plans. If fact you can remove this override and use the function in the base class:
https://github.com/cmu-db/peloton/blob/master/src/planner/abstract_plan.cpp#L65